ports/jsoncons/vcpkg.json

```
@@ -1,7 +1,8 @@
 {
   "name": "jsoncons",
-  "version-string": "0.168.3",
+  "version": "0.168.7",
   "port-version": 1,
```

```suggestion
  "version": "0.168.7",
```
Thanks for your PR! Can you please correct the Version field in the CONTROL file? You can get more information from [maintainer-guide.md](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md).